### PR TITLE
Add automate docs

### PR DESCRIPTION
**Pull request by IanMadd**
### Description

- Adds Automate docs to docs.chef.io
- Release Notes:
   - creates a new menu section for release notes
   - adds release notes for Automate
   - adds release notes for InSpec
   - migrates other release notes from Chef Infra menu section to the Release Notes section
- Updates Expeditor config for Automate
- Updates links in chef-web-docs to Automate docs
- Fixes a few menu links that changed when sites were consolidated to community.chef.io

### Issues Resolved

chef/release-engineering#1239

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by kagarmoe** [comment]
Waiting for the build...
tenor-43401373

**Review by mjingle — APPROVED** [review]
tenor-168214626

**Review by kagarmoe — APPROVED** [review]
tenor-183265689

File: .expeditor/config.yml [hunk 1]
```diff
@@ -22,3 +22,6 @@ subscriptions:
   - workload: artifact_published:stable:inspec:*
     actions:
       - bash:.expeditor/update_hugo_modules.sh
+  - workload: artifact_published:stable:automate:*
```
Comment (schisamo) [inline .expeditor/config.yml]: The "stable" channel for Automate is actually `current`. Additionally Automate does not promote an Omnibus package so the workload we need to subscribe to is slightly different:
```
# Subscribe to the promotion of the acceptance channel to current
workload: project_promoted:automate:acceptance:*
```

**Review by schisamo — CHANGES_REQUESTED** [review]

**Review by schisamo — APPROVED** [review]

**Pull request merged**